src/decode.h

```diff
@@ -411,6 +411,8 @@ enum PacketDropReason {
     PKT_DROP_REASON_STREAM_ERROR,
     PKT_DROP_REASON_STREAM_MEMCAP,
     PKT_DROP_REASON_STREAM_MIDSTREAM,
+    PKT_DROP_REASON_STREAM_REASSEMBLY,
+    PKT_DROP_REASON_MAX,
```

Comment:
It accidentally made it into this branch too.

Reply:
:-(